code/__DEFINES/cards.dm

```diff
@@ -0,0 +1,17 @@
+#define CREATURE_CARD "CREATURE"
+#define EFFECT_CARD "EFFECT_CARD"
+#define EQUIPMENT_CARD "EQUIPMENT_CARD"
+#define AREA_CARD "AREA_CARD"
+
+//These are rarity weights, the higher the number the more common it is in booster packs
+#define COMMON		100
+#define UNCOMMON	75
+#define RARE 		50
+#define LEGENDARY	5
+#define ROBUST		1
+#define ADMEME		0
```

Comment:
Mono red player: "I cast smite the monstrous!"
Mono blue player: "I cast negate, then cast negate to counter my negate to trigger this effect which counters your ability and untaps all of my creatures"